scene/2d/canvas_item.cpp

```diff
@@ -1075,6 +1075,7 @@ Vector2 CanvasItem::make_canvas_position_local(const Vector2 &screen_point) cons
 Ref<InputEvent> CanvasItem::make_input_local(const Ref<InputEvent> &p_event) const {
 
 	ERR_FAIL_COND_V(!is_inside_tree(), p_event);
+	ERR_FAIL_COND_V(p_event.is_null(), NULL);
```

Comment:
It's mostly cosmetic, but I'd say you could return `p_event` here to be consistent with the previous line and overall logic: return the input on failure. Even though here `p_event` *is* NULL so it's the same in the end.